docs/ReleaseProcess.md

```diff
@@ -0,0 +1,89 @@
+# Release Process: openlaw-core
+
+Note this documentation is for core project maintainers and all steps may not apply to open source contributors. If in doubt, just ask! We're friendly folks. :rainbow:
+
+## Submitting Changes :sparkles:
+
+New code changes should always be introduced via a Pull Request branch:
+
+1. Pull latest `master` branch then create new branch off from `master` and make the changes there.
+
+2. If added any new sbt tests, make sure they are all passing.
+
+3. Submit PR for merging that new branch into `master`. GitHub Actions will automatically attempt a build and run tests, which you can see here: https://github.com/openlawteam/openlaw-core/actions.
+
+4. If the build and all checks are passing, the PR can be merged to master.
+
+## Releasing to BinTray :rocket:
+
+The `master` branch represents the current stable set of the code, and should be considered "release-able" at any point in time.
```

Comment:
should we also add - "Please don't merge anything into `master` until you feel it's production-ready."?

Reply:
I think it's more that we need to make sure that:
1. all the tests pass (no force merge there!) 
2. that we have done proper code review to make sure it is up to our standards. 
3. I think that the point in the document saying that it should be "release-able" at any point in time is a good way of summarizing all that.

In the future we should spend some time defining our definition of done for a task